frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -139,7 +139,7 @@
         }
     }
 
-    @each $status in ('default', 'primary', 'danger', 'primary-alt') {
+    @each $status in ('default', 'primary', 'danger', 'primary-alt', 'muted') {
```

Comment:
Muted was purposefully removed. @clarkus made changes such that there should be no use case for that button style.

Reply:
🤔 It's outright used in the annotations designs I'm working with though @clarkus Should `muted` stay after all or does it need to be replaced in the mocks?

Reply:
Btw I think the context here was it wasn't very accessible-friendly regarding color differentiation which is why we got rid of it.

Reply:
Clarified this in slack, but it's muted-alt that is being deprecated. Text-muted / secondary text is still fine to use.

Reply:
Okie dokie. I feel like we put a fair amount of effort into the discussion that led us to remove `muted` as well but if you say 👍  then I say 👍 

@Twixes I will create a PR to add this to storybook as well

Reply:
To be clear, it was always the muted-alt (#747EA1) that I was referring to. The other text-muted is still used. Sorry for the confusion - let me know if I can clarify this more in documentation / figma.